src/server/ftpserver/options.rs

```diff
@@ -240,3 +240,16 @@ impl Default for FailedLoginsPolicy {
         FailedLoginsPolicy::new(3, Duration::from_secs(120), FailedLoginsBlock::UserAndIP)
     }
 }
+
+/// The options for [Server.data_connection_mode](crate::Server::data_connection_mode).
+/// This allows to switch active / passive mode on or off.
+#[derive(Debug, PartialEq, Eq, Clone, Copy, Default)]
+pub enum DataConnectionMode {
```

Comment:
Alternative name is `ActivePassiveMode` which is probably more clear. What do you think @robklg ?

Reply:
I agree `ActivePassiveMode` is clearer.

Reply:
Renamed